src/driver/driver.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+require('dotenv').config();
+const io = require('socket.io-client');
+const port = process.env.config || 3333;
```

Comment:
This line is causing issues, it needs to be `process.env.PORT` because what is happening both driver and vendor are on port 3333 and caps is on 3000 so they aren't talking to each other which means when I run your code I see no messages.